README.md

```diff
@@ -378,6 +379,47 @@ You can use the following command to vet, format, lint, and test your code:
 $ make test
 ```
 
+### Remote Debug Kogito Operator using Intellij IDEA
+
+The operator will be deployed over an Openshift or Kubernetes cluster. In this example we've taken minikube to deploy a Kubernetes cluster locally.
+
+**Install Minikube:**
+
+For installing the Minikube cluster please follow [this tutorial](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+```bash
+$ minikube start
+```
+**Apply Operator manifests:**
+
+```bash
+$ export NAMESPACE=default
+$ ./hack/install-manifests.sh
+```
+
+**Start Operator in remote debug mode:**
+
+```bash
+$ cd cmd/manager
+$ export WATCH_NAMESPACE=default
+$ dlv debug --headless --listen=:2345 --api-version=2
```

Comment:
from where this command is coming from? go-delve/delve?
might be good to specify it.

Reply:
done